### PR TITLE
Update the Gemfile.lock by reinstalling all gems

### PR DESCRIPTION
**Pull request by digitalronin**
The dependabot PR to bump the version of rubyzip (#73) failed
because of dependency issues.

For this commit, I removed the Gemfile.lock altogether, and ran
`bundle install` to recreate it.

This version has rubyzip 2, as per #73, but doesn't have any
dependency problems, AFAICT.

Running `bundle exec jekyll serve --watch`, the documentation site
worked fine on my laptop.

closes #73

### CODE REVIEW

**Comment by digitalronin** [comment]
This needs a ruby version bump.

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,8 @@
+<<<<<<< HEAD
 rvm: 2.3.3
+=======
+rvm: 2.5.3
+>>>>>>> f149fe0... Change ruby version to 2.5.3
```
Comment (SteveMarshall) [inline .travis.yml]: I don’t think you intended to have diff markers in this commit?

Comment (digitalronin) [reply .travis.yml]: No, sorry. I jumped down a rabbit hole. Fixing now.

**Comment by digitalronin** [comment]
Closing in favour of #75

**Pull request closed without merging**